### PR TITLE
Fixes #1227: Mobile Data Download issue fixed.

### LINKED ISSUE #1227: Add Option to download event information on mobile data
**Actual Behaviour**

Right now data can't be downloaded on mobile data

**Expected Behaviour**

Since almost everyone uses mobile data today, it is imperative to have the option to download data on mobile data

### PR DESCRIPTION
**Pull request by betterclever**
Fixes #1227: Data can now be downloaded over mobile network. 

Changes: Condition to check mobile data working and download over it was not correctly enforced. Fixed that. 

Screenshots for the change: N/A

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 1]
```diff
@@ -209,7 +209,13 @@ public void onClick(DialogInterface dialogInterface, int i) {
                                         OpenEventApp.postEventOnUIThread(new DataDownloadEvent());
                                         sharedPreferences.edit().putBoolean(ConstantStrings.IS_DOWNLOAD_DONE, true).apply();
 
-                                    } else {
+                                    }
```
Comment (harshitagupta30) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Can you please reformat the code, the part which you've added?

**Comment by betterclever** [comment]
@harshitagupta30 Done.

**Comment by betterclever** [comment]
@manonwason Done.

**Comment by betterclever** [comment]
@mananwason Please review.

**Comment by betterclever** [comment]
DO NOT MERGE. I realized that user has option to make choice to opt out of mobile data. That choice is not being respected in this PR. I'm amending commit.

**Pull request closed without merging**